components/ProjectModal/styles.ts

```diff
@@ -34,12 +35,10 @@ export const ProjectDetails = styled.div`
   height: 100%;
   background: ${COLORS.white};
   width: 340px;
+  overflow-y: auto;
```

Comment:
I think you also need to add `overflow-x: hidden`
Currently when I scroll horizontally on the project modal it scrolls.
<img width="428" alt="Screenshot 2024-11-04 at 12 46 49 AM" src="https://github.com/user-attachments/assets/afd7f5a0-68c4-47fc-b119-42eef1cec637">